pom.xml

```diff
@@ -82,6 +82,21 @@
   </scm>
   <build>
     <plugins>
+      <plugin>
+        <groupId>com.mycila</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <configuration>
+          <licenseSets>
+            <licenseSet>
+              <excludes>
+                <exclude>**/LoginExtension\.js</exclude>
+                <exclude>**/GoogleAppsConfigClass\.js</exclude>
+                <exclude>**/GoogleAppsConfigSheet\.vm</exclude>
```

Comment:
Why not add the missing license header? See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-tree/xwiki-platform-tree-webjar/src/main/webjar/tree.js#L1-L19 and https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/html_displayer/date/edit.vm#L1-L19 .

Reply:
The way they are used right now is that they are injected into the content tag of the xml pages with the same name, at build time. I was thinking that it was unnecessary to also have license header for these files, given the way they are used.